jupyter_ui_poll/_poll.py

```diff
@@ -44,6 +45,12 @@ def __init__(self, shell, loop) -> None:
         )
         self._events: List[Tuple[Any, Any, Any]] = []
         self._backup_execute_request = kernel.shell_handlers["execute_request"]
+        self.backup_main_asyncio_lock = None
```

Comment:
Can you please use leading `_` for consistency:
```suggestion
        self._backup_main_asyncio_lock = None
```